go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	gomodules.xyz/jsonpatch/v3 v3.0.1
```

Comment:
It is missing the fragment file. Could you please add it? See : https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments

You need copy a template and add the info for this one, such as:

```yaml
# entries is a list of entries to include in
# release notes and/or the migration guide
entries:
  - description: >
     Upgrade helm dependency for Helm based-Operators from `v3.2.0` to `v3.2.4` in order to fix fix CVE-2020-4053.
    # kind is one of:
    # - addition
    # - change
    # - deprecation
    # - removal
    # - bugfix
    kind: "change"
    # Is this a breaking change?
    breaking: false
```